CHANGELOG.md

```diff
@@ -1,3 +1,190 @@
+## Release 0.14.0
+
+Highlights:
+
+This release comes with two main new features:
+  * Plugins. Now you can create your http-based plugin to provide new metrics
+    and display them in Scope. You can read more about it and see some examples [here]().
+  * _Metrics on canvas_: Metrics are now displayed on the nodes and not just on
+    the details panel, starting with CPU and memory consumption.
+
+Also, the performance of the UI has been considerably improved and the 100-node
+rendering limit has been lifted.
+
+
+New features and enhancements:
+- Metrics on canvas
+	[#1105](https://github.com/weaveworks/scope/pull/1105)
+	[#1204](https://github.com/weaveworks/scope/pull/1204)
+	[#1225](https://github.com/weaveworks/scope/pull/1225)
+- Scope Plugins
+	[#1126](https://github.com/weaveworks/scope/pull/1126)
+	[#1277](https://github.com/weaveworks/scope/pull/1277)
+	[#1280](https://github.com/weaveworks/scope/pull/1280)
+	[#1283](https://github.com/weaveworks/scope/pull/1283)
+- Node details panel improvements
+  - Add connection tables
+	[#1017](https://github.com/weaveworks/scope/pull/1017)
+	[#1248](https://github.com/weaveworks/scope/pull/1248)
+  - Layout: make better use of column space
+		[#1272](https://github.com/weaveworks/scope/pull/1272)
+  - Sparklines
+    - Update every second and show 60sec history
+			[#795](https://github.com/weaveworks/scope/pull/795)
+    - Apply format to tooltips in hovers
+			[#1230](https://github.com/weaveworks/scope/pull/1230)
+  - Sort numerical entries (e.g. image counts, Process Ids) as expected
+		[#1125](https://github.com/weaveworks/scope/pull/1125)
+  - Remove load5 and load15 metrics
+		[#1274](https://github.com/weaveworks/scope/pull/1274)
+- Graph view improvements
+  - Node filtering improvements
+      - Introduce three-way filtering selectors (e.g. choose from `System containers`, `Application containers` or `Both`)
+			[#1159](https://github.com/weaveworks/scope/pull/1159)
+      - Maintain node-filtering selection across subviews (e.g. `Containers by ID` and `Containers by Image`)
```

Comment:
Monospace text might not be the best choice for these enumerations, as they're not code... how about bold?